src/commands/ReminderCommand.ts

```diff
@@ -95,13 +132,31 @@ class ReminderCommand extends Command {
 						setTimeout(() => this.elapse(interaction.client, id), milliseconds)
```

Comment:
Oh I see now, this should be 
```suggestion
						setTimeout(() => this.elapse(interaction.client, id), timestamp - Date.now())
```

Reply:
Thanks :)